lunes_cms/cms/admins/feedback_admin.py

```diff
@@ -21,12 +21,37 @@ class FeedbackAdmin(admin.ModelAdmin):
     search_fields = ["comment"]
     list_filter = ["content_type", "read_by"]
     sortable_by = ["content_type", "created_date", "read_by"]
+    actions = ["mark_as_read", "mark_as_unread"]
 
     def has_add_permission(self, request, obj=None):
         return False
 
     def has_change_permission(self, request, obj=None):
         return False
 
+    @admin.action(description=_("Mark as read"))
+    def mark_as_read(self, request, queryset):
+        """
+        Action to mark selected items as read by user
+
+        :param request: current user request
+        :type request: django.http.request
+        :param queryset: current queryset
+        :type queryset: QuerySet
+        """
+        queryset.update(read_by=request.user)
+
```

Comment:
```suggestion
        messages.success(
            request,
            _(
                "The selected feedback entries were successfully marked as read.",
            ),
        )

```
Maybe add a small note here? (this requires to import `from django.contrib import messages`)